src/main/java/com/avengers/gamera/service/ArticleService.java

```diff
@@ -93,15 +92,14 @@ public String deleteArticleById(Long articleId) {
        return "The article with ID("+ articleId +") has been deleted";
     }
 
-    public ArticleGetDto updateArticle( Long articleId,ArticlePostDto articlePostDto){
+    public ArticleGetDto updateArticle (ArticlePutDto articlePutDto){
+        Long articleId = articlePutDto.getArticleId();
+        Article article = articleRepository.findById(articleId).orElseThrow(() -> new ResourceNotFoundException("Article",articleId));
```

Comment:
The below method is better. If the article is deleted, normal editor cannot get it anymore